### ISSUE TITLE
MuseScore Studio 4.4.2 crashes when trying to Play a score(s) [both Windows 10 & 11]

### ISSUE THREAD

**Issue by VladaScore** [musescore/MuseScore#24956]
### Issue type

Crash or freeze

### Description with steps to reproduce

1. I create a brand new score and MuseScore plays it fine.
2. I save that score, close it and quit MuseScore.
3. I start MuseScore and open the same score.
4. When pressing the Play button MuseScore crashes :-(

So, everything works fine while a brand-new score is created and being played (before closing it). MSStudio plays it fine only and only until that score file gets closed. Furthermore, after pressing Play button MSStudio crashes without playing any single note.

My MuseScore Studio 4.4.2 also crashes when trying to play any of the old scores.

Please note that I use Native Instruments - Kontakt 7 and Virtual Accoustic VST accordion. I have been using this software for more than a year and everything worked just fine with the MuseScore Studio 4.3.x.

### Supporting files, videos and screenshots

[Diagnostics_Win-11_MuseScore-4.4.2-Crashes.zip](https://github.com/user-attachments/files/17172239/Diagnostics_Win-11_MuseScore-4.4.2-Crashes.zip)
[Diagnostics_Win-10_MuseScore-4.4.2-Crashes.zip](https://github.com/user-attachments/files/17172245/Diagnostics_Win-10_MuseScore-4.4.2-Crashes.zip)
[https://musescore.org/en/node/369488](url)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 10 Home and Windows 11 Enterprise

### Additional context

My MuseScore Studio 4.4.2 also crashes when trying to play any of the old scores, both on Windows 10 Home and Windows 11 Enterprise :-(

a) I have already reinstalled my MuseScore 4.4.2 Studio and it didn't help.

b) Reverting MuseScore to factory settings doesn't help either.

c) Important or not, but I have no ideas about the errors: "Could not find the tokens file: C:/Users/Kw5U/AppData/Local/MuseScore/MuseScore4/musescorecom_cred.dat and audiocom_cred.dat".

d) I also tried to bypass the Log error message: "ERROR | main_thread | ExtensionsConfiguration::manifestConfigs | failed read config data, err: [406] An error occurred when reading from the file: C:/.../AppData/Local/MuseScore/MuseScore4/plugins/plugins.json".
I temporarily created a directory ...\AppData\Local\MuseScore\MuseScore4\plugins and copied file ...\AppData\Local\MuseScore\MuseScore4\known_audio_plugins.json as ...\AppData\Local\MuseScore\MuseScore4\plugins\plugins.json.
This doesn't help.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by VladaScore** [comment]
My apology, I forgot to mention above that there is a lot of QML error messages in the MS Studio logs.

**Comment by cbjeukendrup** [comment]
In both attached crash logs, the crash is caused by Kontakt. We'll need to look for a way to work around the crash.

**Comment by VladaScore** [comment]
Casper, thank you for the quick action and your investigation, but last time I have to point out and stress it that both Music Score Studio and Kontakt work perfecly together with the brand new score without any single mulfunction, failior or crash.
Once the score file gets closed and that session ends, crashes or problems start when the same (or any previous) music score gets reloded !

Personally I beleive that there might be a problem in the interaction between the Kontakt software and MuseScore 4.4.x.
I beg you to check it and re-think about the data and/or requests MuseScore saves into .mscz file and later sends it to Kontakt, please. There must be something wrong there, as no single crash ever happened in Muse Score Studio 4.3.x with the same Kontak.

FYI, there is one additional VST related malfunction that I will report later within the VST issues. It is, most likely, corelated with these Studio crashes.

**Comment by DmitryArefiev** [comment]
@VladaScore Did you get a crash with other VSTi using Kontakt 7? (maybe with some Native Instruments libraries)

(I don't have Virtual Accoustic VST accordion and there is no trial of it)

**Comment by cbjeukendrup** [comment]
@VladaScore Yes, I understood that already. There is doubtlessly some way that we can work around it, and doubtlessly some reason that it didn't happen before (for example, maybe we didn't use a certain functionality of the VST API), but a fact is that the actual crash happens inside Kontakt code. But as I said, we are indeed going to look into it.

**Comment by VladaScore** [comment]
@DmitryArefiev  Unfortunately, I only use VA Guerrini Accordion, but I might be able to try to reproduce the crash with Efimov Bayan for the KONTAKT. If manage to do it, I will provide you the results. OK?
![VST-Efimov-Bayan](https://github.com/user-attachments/assets/c174efbf-1984-465b-bc47-57c3a2a1e31a)

**Comment by VladaScore** [comment]
@cbjeukendrup Thank you so much!
As I explained in #24980, the root cause of both issues might be the special control notes (keys).
![KONTAKT-VST-Guerrini-Accordion](https://github.com/user-attachments/assets/727572ae-50d7-4086-9ca5-dbff862b9ebf)

**Comment by VladaScore** [comment]
@DmitryArefiev MS4.4 also crashed with the VST Bayan. The diagnostic file is attached.
[MSStudioCrash_Widows-11_VST-NI-Kontakt-BayanAccordion.zip](https://github.com/user-attachments/files/17199380/MSStudioCrash_Widows-11_VST-NI-Kontakt-BayanAccordion.zip)

**Comment by DmitryArefiev** [comment]
@VladaScore Thanks for the new crashlogs. Unfortunately, VST Bayan is also not available in demo version. 
I have KOMPLETE 13 ULTIMATE Collectors Edition, will try to find some instrument with key switches there.

**Comment by VladaScore** [comment]
@DmitryArefiev My apology for the delay. Unfortunately, I don't have NI-KOMPLETE software :-(
By the way, I would have never used KONTAKT either if I could find better accordion samples. 
Hopefully, Muse Hub will offer (a) good, professional accordion(s) one day for a decent price :-)

**Comment by DmitryArefiev** [comment]
@VladaScore We don't have licenses yet to understand the issue properly.. But if you don't have any alternative accordion VST  that works with 4.4.x, you can use 4.3.2 build in portable version to work with that specific VST. You can download it here https://github.com/musescore/MuseScore/releases (use MuseScore-Studio-4.3.2.241630831-x86_64.paf.exe )

**Comment by VladaScore** [comment]
@DmitryArefiev Thank you Dmitry so much for not forgetting me :-)
VST accordion is the musical instrument No1 in all scores of mines. 
Personally, I bypassed the trouble caused by rollout of the MuseScore Studio 4.4.x by restoring a system backup altogether with the OS. Otherwise, it was mission impossible because Studio 4.4.x doesn't play the new MuseSound fonts and the installation of new MuseHub removed old SoundFonts. Old version of the MuseHub is not available for download anymore and the old fonts were kept in the hidden system directory ProgramData, etc. So, the average user is stuck with new versions of MuseHub and MuseScore which is not good because I am not the only one experiencing Studio crashes related to the "VST KONTAKT bug(s)".
Thanks anyway and I will be happy to see Italian Accordion Guerrini, Scandalli, Settimio Soprani, Bugari or Beltuna offered by MuseHub one day - the sooner, the better :-)
Best regards!

P.S. By the way, FYI MS Studio 4.4.x also crashes with the newest KONTAKT 8 .

**Comment by CubikingChill** [comment]
@VladaScore Please have a look at my fix #16144 if the issue is still there.
